src/cogniteClient.ts

```diff
@@ -386,6 +386,10 @@ export default class CogniteClient {
   public delete = <T = any>(path: string, options?: HttpRequestOptions) =>
     this.httpClient.delete<T>(path, options);
 
+  public setNextRequestSdkHeader(value: string) {
```

Comment:
I think a better name could be `setOneTimeHeader`

Reply:
I've named it `setOneTimeSdkHeader` now, since it's only about that one specific header